docker/rootfs/usr/local/bin/run_worker.sh

```diff
@@ -18,5 +18,5 @@ fi
 if [ "$#" -gt 0 ]; then
     shift
 fi
-exec ${MAYAN_PYTHON_BIN_DIR}celery -A mayan worker -Ofair -l ERROR -Q ${MAYAN_QUEUE_LIST} --without-gossip --without-heartbeat ${@}
+exec ${MAYAN_PYTHON_BIN_DIR}celery -A mayan worker -Ofair -l ${MAYAN_WORKER_LOG_LEVEL} -Q ${MAYAN_QUEUE_LIST} --without-gossip --without-heartbeat ${@}
```

Comment:
it would be great to have all the env variables for debugging listed on a confluence page